chef/cookbooks/crowbar/templates/default/crowbar-evacuate-host.sh.erb

```diff
@@ -41,17 +41,20 @@ blockmigrate=""
 <% end %>
 if ! nova host-evacuate-live "$host" $blockmigrate; then
     log "Live migration of instances from host $host has failed!"
-    touch $UPGRADEDIR/crowbar-evacuate-host-failed
+    echo 2 > $UPGRADEDIR/crowbar-evacuate-host-failed
     exit 2
 fi
 
-# FIXME: here we need checks for potential errors during the evacuation
-
 while nova list --all-tenants --host "$host" --fields host,status 2>/dev/null | grep -e ACTIVE -e MIGRATING | grep -q "$host"; do
     log "There is still some VM running at $host..."
     sleep 10
 done
 
+if nova list --all-tenants --host "$host" --fields status | grep -q ERROR; then
```

Comment:
> What happens if the customer had an instance in error status from some time ago and never 
> cleaned it up?

Good point, but actually I think now user has time to clean it up and restart the step until there is not error instance.

> having an instance on error does not necessarily mean that the migration failed

Hm, true. Any idea for better check? Should we have for "Power State" being migrating? (I'll have to test it again...)